src/sdg/templates/core/_siteconfig.html

```diff
@@ -0,0 +1,4 @@
+{% if siteconfig.goatcounter_code %}
+    <script data-goatcounter="https://{{ siteconfig.goatcounter_code }}.goatcounter.com/count"
+            async src="//gc.zgo.at/count.js"></script>
```

Comment:
It works slightly different:

```suggestion
    <script data-goatcounter="https://{{ siteconfig.goatcounter_domain }}/count"
            async src="//{{ siteconfig.goatcounter_domain }}/count.js"></script>
```

Please adjust the model fields and code.